src/service/inputs/parsed-inputs.ts

```diff
@@ -0,0 +1,25 @@
+import { defaultInputValues, InputValues, LoggerLevel } from "@bc/domain/inputs";
+import { Service } from "typedi";
+
+@Service()
+export class ParsedInputs {
```

Comment:
this class name is very confusing, it took me a while to figure out this is actually a service, the name FMPOV means more a POTO (Plain Old Typescript Object) than a service, it looks like ParsedInputs is a plain domain object, what if we call this `InputService`